### ISSUE TITLE
[Feature]: Bridges

### ISSUE THREAD

**Issue by trymeouteh** [simplex-chat/simplex-chat#3684]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

all

### App version

5.4.2

### Feature

Matrix has bridges which allows matrix rooms to connect to other messaging platforms and protocols. Would be a cool feature to see this for SimpleX

Bridges to connect SimpleX groups to Discord, Matrix, XMPP, etc.

This will likely require a new app that will act as a bridge. It will connect to SimpleX groups and groups on other platforms and bridge the two worlds togeather. The app could be headless or have a GUI, allow it to be ran on servers or desktops. It will have the SimpleX group chat features built in and will not support any other platforms. 3rd party developers will be able to create plugins (bridges) which can be downloaded and updated in the app from github, gitlab, gitea, forgejo and then setup by the user. No need for the SimpleX devs to build bridges for each platform.

Features can include...

- admin controls (allow admins on 3rd party platforms ban users in SimpleX group or allow SimpleX group admin ban users on 3rd party platform group)

A feature should be implimented in SimpleX informing users if there are bridges being used in a SimpleX group chat for transaprency on the group chat privacy.

**Comment by erebion** [comment]
As someone who primarily uses Matrix, I'd also like that. I enjoy having everything in one client and I would definitely adopt Simplex in an instant if I could just set up such a bridge.

Right now, I don't really use Simplex, simply because my use-case requires multi-device support and some other stuff, however, with a bridge I get everything I'm used to with Matrix while being able to communicte to users that use Simplex.

I already run some other bridges to make my life easier, especially for services that only allow to be used on one device, which is always a pain. But with a bridge I can use as many devices as I want, as the bridge then is the only device. :)

**Comment by Sepero** [comment]
I'm not sure if this is compatible with the SimpleX anonymity structure. All those other networks have User Identification.

**Comment by ilmaisin** [comment]
I agree that using SimpleX as a some sort of personal multi-platform chat hub won't make much sense.

However, one could maybe want to use something like [matterbridge](/42wim/matterbridge) to relay messages between group chats on SimpleX and other platforms.

**Comment by erebion** [comment]
> I agree that using SimpleX as a some sort of personal multi-platform chat hub won't make much sense.

No, but it could be used via a local XMPP gateway, perhaps.

**Comment by ikkqqstudio** [comment]
+1
